Examples/Tests/ohm_solver_ion_beam_instability/PICMI_inputs.py

```diff
@@ -368,7 +371,7 @@ def text_diag(self):
             "{step_rate:4.2f} steps/s"
         )
 
-        if sim_ext.getMyProc() == 0:
+        if libwarpx.amr.ParallelDescriptor.MyProc() == 0:
```

Comment:
These checks for the root proc can also be done as:
```
if simulation.extension.warpx.getMyProc() == 0:
```
which would avoid the need to `import libwarpx` in several of the changed files.